src/core/styles/common/mixins/body.ts

```diff
@@ -0,0 +1,9 @@
+import { css } from "@emotion/react";
+import { FontWeights } from "../constants/fontWeights";
+
+export const fontXs = css`
```

Comment:
@hthomas-czi this is an example of defining a mixin!

Reply:
And this is how to access the theme object in Emotion 😄 

```ts
const StyledAlert = styled(Alert)`
  ${shadows}
  ${palette}
  ${({ theme }) => {
    console.log("!!!! GOT THEME", theme);
    return `
      background-color: ${theme.palette.success.light}
    `;
  }}
`;
```

https://codesandbox.io/s/material-demo-forked-jkb49?file=/demo.tsx:782-988

<img width="1443" alt="Screen Shot 2021-03-10 at 2 00 39 PM" src="https://user-images.githubusercontent.com/6309723/110703860-21b68d00-81a9-11eb-963a-cbc59b1a0f60.png">